spec/serializers/project_serializer_spec.rb

```diff
@@ -31,6 +32,43 @@
         stars
         status
       ]
-    )
+                                         )
+    end
+  end
+
+  context "with updated_at" do
+    subject { described_class.new(build(:project), show_updated_at: true) }
+
+    it 'should have expected attribute names' do
+      expect(subject.attributes.keys).to eq(
+        %i[
```

Comment:
Maybe pull the non-internal ones into a shared constant so the list isn't totally duplicated and it's clear what the difference between them is?